modules/02-cdc-connect-debezium.md

```diff
@@ -85,7 +92,7 @@ Run an Apache Kafka console consumer on one of the pods for receiving messages f
 
 ```shell
 export CONSOLE_CONSUMER_PASSWORD=$(oc get secret kafka-console-consumer -o jsonpath='{.data.password}' | base64 -d)
-oc exec -it my-cluster-kafka-0 -- /opt/kafka/bin/kafka-console-consumer.sh \
+oc exec my-cluster-kafka-0 -c kafka -- /opt/kafka/bin/kafka-console-consumer.sh \
```

Comment:
Why you removed "-it". It's needed to avoid that doing CTRC + C it doesn't stop the related process in the container and the console consumer is still alive inside it.